### PR TITLE
[feat] Update alerts for Cozy Files v3

### PR DESCRIPTION
**Pull request by gregorylegarec**
* Adds styles for an overlay
 * Remove styles for icons in alerts
 * Update margin/padding, etc.
 * Adds styles for buttons in alerts
 * Deal with CSS transition on overlay and alerts

### CODE REVIEW

File: stylus/ui-app/alerts.styl [hunk 1]
```diff
@@ -21,23 +31,46 @@ $alerts
...
+        top                        -25%
+        opacity                    0
+        transition-timing-function ease-in
+
+      .coz-overlay
+        opacity 0
+
+    // Deprecated update existing code and remove
+    // User .coz-alerter--hidden on .coz-alerter instead
     .coz-alert--closing
```
Comment (CPatchane) [inline stylus/ui-app/alerts.styl]: I see that you replaced `.coz-alert--closing` by `coz-alerter--hidden`. @GoOz How do we handle changes in cozy-ui that should be updated in other app (here just for cozy-settings)? A changelog?

Comment (GoOz) [inline stylus/ui-app/alerts.styl]: Well we should limit changes as much as possible, but if we have to a changelog and an update on the version should be done, obviously. That being said, we're still on the early stage of Cozy-UI. I think we can skip on this for the time being, until we have a stable Cozy-UI repo. Don't you think? /poke @m4dz

File: stylus/ui-app/alerts.styl [hunk 2]
```diff
@@ -21,23 +31,46 @@ $alerts
...
         .coz-alert-title
             font-weight bold
 
     .coz-alert--error
         background-color pomegranate
-        background-image embedurl('../../assets/icons/icon-warn.svg')
 
+        .coz-btn
+          color         pomegranate
+          border        none
```
Comment (GoOz) [inline stylus/ui-app/alerts.styl]: If you have a new style for a button, you should put it in `/ui-components/button.styl`
And giving the fact that a style for .coz-btn already exists, if you can't find a matching style on the already existing modifiers, just add yours there.

Comment (gregorylegarec) [inline stylus/ui-app/alerts.styl]: How do you think I should proceed ?
```css
.coz-alert .coz-btn {}
// or
.coz-alert-btn {}
```

File: stylus/ui-app/alerts.styl (hunk 1 above)
Comment (CPatchane) [inline stylus/ui-app/alerts.styl]: @gregorylegarec I suggest for now to remove the old way and I will update cozy-settings to use the new way. We will discuss about the cozy-ui changelog later.

File: stylus/ui-app/alerts.styl (hunk 2 above)
Comment (CPatchane) [inline stylus/ui-app/alerts.styl]: I have a personal preference for the first one which uses the same global .coz-btn class

File: stylus/ui-app/alerts.styl [hunk 3]
```diff
@@ -39,41 +39,23 @@ $alerts
         cursor                     pointer
 
         p
-          margin 0
-          line-height 1.5
+            margin 0
```
Comment (GoOz) [inline stylus/ui-app/alerts.styl]: it just needs aligned values and we're good :)

Comment (gregorylegarec) [inline stylus/ui-app/alerts.styl]: Oops, my bad :3

**Comment by GoOz** [comment]
Thanks 😉

**Pull request merged**